src/include/ref_inline.h

```diff
@@ -0,0 +1,118 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+/*
+ * __wt_ref_is_root --
+ *     Return if the page reference is for the root page.
+ */
+static WT_INLINE bool
+__wt_ref_is_root(WT_REF *ref)
+{
+    return (ref->home == NULL);
+}
+
+/*
+ * __wt_ref_set_state --
+ *     Set a ref's state. Accessed from the WT_REF_SET_STATE macro.
+ */
+static WT_INLINE void
+__wt_ref_set_state(WT_REF *ref, uint8_t state)
```

Comment:
Yeah I changed it since we had calls to the function outside of ref_inline.h and pattern matched on our style guide.
I've rolled this back in [e21f8c0](https://github.com/wiredtiger/wiredtiger/pull/10345/commits/e21f8c011ebd1f98f34215782f59dece563afacc) and changed those external calls to use the macro instead.